pomodoro-client.py

```diff
@@ -157,9 +181,13 @@ def handle_state(state, old_state):
     else:
         start_dunst()
 
+
 def show_message(message, is_error=False):
+    nagbar = ["swaynag", "i3-nagbar"]
     type_ = "error" if is_error else "warning"
-    Popen('i3-nagbar -t %s -m "%s"' % (type_, message), shell=True)
+    for command in nagbar:
+        Popen('%s -t %s -m "%s"' % (command, type_, message), shell=True)
```

Comment:
This will purposefully launching a command that will fail, right?

Regardless, I'm thinking this could be extracted to a separate open source library, called sth along the lines of `sensible-nag`, which would be just a shell script that opens a nagbar equivalent matching the environment, similarly how `i3-sensible-terminal` opens the "best" terminal you have.

Again, this doesn't need to be the scope of this MR of course :smile_cat:

Reply:
Yes, true, will try to handle this better. This try trigger the two commands in the same time but should only try to start the one available 👍🏻

Reply:
Its fix now, we choose the nagbar depending on whether i3 running or not.